.github/workflows/validate.yml

```diff
@@ -11,58 +11,54 @@ jobs:
     strategy:
       matrix:
         BUILD_TYPE: [Release, Debug]
+        DISTRO: ['fedora', 'fedora_rawhide', 'fedora_mpich', 'opensuse', 'ubuntu', 'ubuntu_devel', 'ubuntu_rolling', 'ubuntu_mpich']
+        CXX: ['g++', 'clang++']
```

Comment:
Can we get `icpc` also into this list? Can we install the Intel toolsuite for all distributions?

Reply:
Somebody could do that, but it won't be me ;-)